src/d3a/d3a_core/area_serializer.py

```diff
@@ -134,6 +134,8 @@ def optional(attr):
         external_connection_available = description.get('allow_external_connection', False)
         baseline_peak_energy_import_kWh = description.get('baseline_peak_energy_import_kWh', None)
         baseline_peak_energy_export_kWh = description.get('baseline_peak_energy_export_kWh', None)
+        import_capacity_kVA = description.get('import_capacity_kVA', None)
+        export_capacity_kVA = description.get('export_capacity_kVA', None)
```

Comment:
Hmm, @spyrostz didnt we discuss in length, to not increase the length of the parameter list of the area?
I reverted some code because of this decision. I think it was done after we had the discussion with Sarah and we split the two Stories (I never should have proposed that, sorry.)

Reply:
I know I saw you coment regarding hte performance on d3a-web, to move the transformer calculation into the d3a.
But still. Next time we have to discuss in more detail, what needs to be done.

Reply:
I do not think there was any other way other than passing more parameters here, if we want to do this calculation on d3a. Moreover, which parameters are you talking about ? If this is about area class, there is a much better workaround in order to decrease the parameter / argument count (stated here https://github.com/gridsingularity/d3a/pull/714#discussion_r407919687). let's discuss it in this comment. 
However, what did the code that you reverted do ?